packages/grid/_modules_/grid/utils/eventEmitter/GridEventEmitter.ts

```diff
@@ -1,15 +1,27 @@
 import * as React from 'react';
 import { EventEmitter } from './EventEmitter';
+import { MuiEvent } from '../../models/muiEvent';
+import { GridCallbackDetails } from '../../models/api/gridCallbackDetails';
 
-// TODO replace any with Generics
-export type GridListener = (params: any, event?: React.SyntheticEvent) => void;
+export type GridValidEvent = MuiEvent<
```

Comment:
What do you think about reusing `MuiEvent`? Two types for events might be confusing.

```tsx
export type MuiEvent<E = React.SyntheticEvent | DocumentEventMap[keyof DocumentEventMap] | {}> =
  E & {
    defaultMuiPrevented?: boolean;
  };
```